core/src/test/java/io/questdb/cutlass/http/DispatcherWriterQueueTest.java

```diff
@@ -203,15 +203,15 @@ public void testAlterTableAddIndexContinuesAfterStartTimeoutExpiredAndTimeout()
                 .withHttpServerConfigBuilder(
                         new HttpServerConfigurationBuilder().withReceiveBufferSize(50)
                 )
-                .withAlterTableStartWaitTimeout(500_000)
-                .withAlterTableMaxWaitTimeout(10_000)
+                .withAlterTableStartWaitTimeout(100)
```

Comment:
500 maybe?

Reply:
this was a broken test, it had been broken on `master` before this PR was merged. The value here is correct to fix the test